django/db/models/sql/datastructures.py

```diff
@@ -70,7 +70,7 @@ def as_sql(self, compiler, connection):
         qn2 = connection.ops.quote_name
 
         # Add a join condition for each pair of joining columns.
-        for index, (lhs_col, rhs_col) in enumerate(self.join_cols):
+        for (lhs_col, rhs_col) in self.join_cols:
```

Comment:
makes sense to me. if `self.join_cols` is a tuple, you don't even need those parentheses.